boards/PDM/Inc/main.h

```diff
@@ -88,78 +82,60 @@ extern "C"
 #define IS_DI_FL_DI_FR_GPIO_Port GPIOC
 #define VBAT_SENSE_Pin GPIO_PIN_0
 #define VBAT_SENSE_GPIO_Port GPIOA
-#define _24V_ACC_SENSE_Pin GPIO_PIN_1
-#define _24V_ACC_SENSE_GPIO_Port GPIOA
-#define _24V_BOOST_OUT_SENSE_Pin GPIO_PIN_2
-#define _24V_BOOST_OUT_SENSE_GPIO_Port GPIOA
-#define PIN_AUX1_Pin GPIO_PIN_3
-#define PIN_AUX1_GPIO_Port GPIOA
-#define PIN_AUX2_Pin GPIO_PIN_4
-#define PIN_AUX2_GPIO_Port GPIOF
-#define CUR_SYNC_AUX1_AUX2_Pin GPIO_PIN_4
-#define CUR_SYNC_AUX1_AUX2_GPIO_Port GPIOA
-#define CSB_AUX1_AUX2_Pin GPIO_PIN_5
-#define CSB_AUX1_AUX2_GPIO_Port GPIOA
-#define FSB_AUX1_AUX2_Pin GPIO_PIN_6
-#define FSB_AUX1_AUX2_GPIO_Port GPIOA
-#define FSOB_AUX1_AUX2_Pin GPIO_PIN_7
-#define FSOB_AUX1_AUX2_GPIO_Port GPIOA
-#define PIN_AIR_SHDN_Pin GPIO_PIN_4
-#define PIN_AIR_SHDN_GPIO_Port GPIOC
-#define PIN_LV_PWR_Pin GPIO_PIN_5
-#define PIN_LV_PWR_GPIO_Port GPIOC
-#define CUR_SYNC_AIR_SHDN_LV_PWR_Pin GPIO_PIN_0
-#define CUR_SYNC_AIR_SHDN_LV_PWR_GPIO_Port GPIOB
-#define CSB_AIR_SHDN_LV_PWR_Pin GPIO_PIN_1
-#define CSB_AIR_SHDN_LV_PWR_GPIO_Port GPIOB
-#define FSB_AIR_SHDN_LV_PWR_Pin GPIO_PIN_2
-#define FSB_AIR_SHDN_LV_PWR_GPIO_Port GPIOB
-#define FSOB_AIR_SHDN_LV_PWR_Pin GPIO_PIN_10
-#define FSOB_AIR_SHDN_LV_PWR_GPIO_Port GPIOB
-#define PIN_DI_BL_Pin GPIO_PIN_11
-#define PIN_DI_BL_GPIO_Port GPIOB
-#define PIN_DI_BR_Pin GPIO_PIN_12
-#define PIN_DI_BR_GPIO_Port GPIOB
-#define SCK_Pin GPIO_PIN_13
-#define SCK_GPIO_Port GPIOB
-#define MISO_Pin GPIO_PIN_14
-#define MISO_GPIO_Port GPIOB
-#define MOSI_Pin GPIO_PIN_15
-#define MOSI_GPIO_Port GPIOB
-#define CUR_SYNC_DI_BL_DI_BR_Pin GPIO_PIN_6
-#define CUR_SYNC_DI_BL_DI_BR_GPIO_Port GPIOC
-#define CSB_DI_BL_DI_BR_Pin GPIO_PIN_7
-#define CSB_DI_BL_DI_BR_GPIO_Port GPIOC
-#define FSB_DI_BL_DI_BR_Pin GPIO_PIN_8
-#define FSB_DI_BL_DI_BR_GPIO_Port GPIOC
-#define FSOB_DI_BL_DI_BR_Pin GPIO_PIN_9
-#define FSOB_DI_BL_DI_BR_GPIO_Port GPIOC
-#define PIN_DI_FL_Pin GPIO_PIN_8
-#define PIN_DI_FL_GPIO_Port GPIOA
-#define PIN_DI_FR_Pin GPIO_PIN_9
-#define PIN_DI_FR_GPIO_Port GPIOA
-#define CUR_SYNC_DI_FL_DI_FR_Pin GPIO_PIN_10
-#define CUR_SYNC_DI_FL_DI_FR_GPIO_Port GPIOA
-#define CSB_DI_FL_DI_FR_Pin GPIO_PIN_15
-#define CSB_DI_FL_DI_FR_GPIO_Port GPIOA
-#define FSB_DI_FL_DI_FR_Pin GPIO_PIN_10
-#define FSB_DI_FL_DI_FR_GPIO_Port GPIOC
-#define FSOB_DI_FL_DI_FR_Pin GPIO_PIN_11
-#define FSOB_DI_FL_DI_FR_GPIO_Port GPIOC
-#define PGOOD_Pin GPIO_PIN_12
+#define FR_STBY_DI_REAR_Pin GPIO_PIN_1
```

Comment:
@akrantz01 is this updated for the new PDM or is this just ported from the old PDM? schematic for the new PDM pin outs could be a little different iirc

Reply:
I'm not super sure. I've been going off a PDF @markomarkusi gave me a few weeks ago

Reply:
can you message joe your email and he'll try to get you set up with altium